src/status_im/contexts/wallet/send/routes/style.cljs

```diff
@@ -12,30 +12,32 @@
   {:flex-direction  :row
    :justify-content :space-between})
 
-(def routes-inner-container
-  {:margin-top      8
+(defn routes-inner-container
+  [first-item?]
+  {:margin-top      (if first-item? 7.5 11)
    :flex-direction  :row
    :align-items     :center
    :justify-content :space-between})
 
-(defn section-label
-  [margin-left]
-  {:flex        0.5
-   :margin-left margin-left})
+(def section-label-right
+  {:width 135})
```

Comment:
why are we hardcoding width? 🤔

Reply:
It is fixed width in Figma. ![](https://github.com/status-im/status-mobile/assets/18485527/965aa1d2-957f-4b16-b3d2-9c06e0fd7068)

Spacing will vary thanks to `:justify-content :space-between` in the container (https://github.com/status-im/status-mobile/blob/ef1c367a8220cbe68eaaa0edd84a7fa1e1234f87/src/status_im/contexts/wallet/send/routes/style.cljs#L13)